data-acquisition/nomask_generator/.ipynb_checkpoints/No_mask_downloader-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we not check in checkpoint files? `.ipynb_checkpoints` should be in `.gitignore`, please add it if it doesn't already exist.

Reply:
Yup, good point. They're removed.